lightshow/postprocess/autobroaden.py

```diff
@@ -0,0 +1,289 @@
+
+"""Using known XAS exprimental spectrum and simulation to optimize the broadening pararmeters for other simulations"""
+
+
+from scipy.stats import pearsonr as ps
+from scipy.interpolate import interp1d
+from scipy.stats import spearmanr
+from scipy import optimize
+from scipy.stats import norm
+from scipy.stats import cauchy
+from scipy.special import voigt_profile as voigt
+
+import numpy as np
+
+
+class Site_spectum:
+    def __init__(
+        self,
+        xin,
+        yin,
+        VASP_out = [float, float, float],
+        weight = 1.0,
+        
+    ):
+        """
+        The spectrum information of a each site
```

Comment:
Docs should be [numpy](https://numpydoc.readthedocs.io/en/latest/format.html) style.